### DISCUSSION TITLE
How to save setconfig settings ?

### DISCUSSION THREAD

**Discussion by tanreeve** [ahrm/sioyek#622] [Q&A] [answered]
Hello, I'm new using this app. 

in the command palette, I typed `setconfig_custom_background_color`#d4b595, but when I restart the app and toggle that custom color , that custom color doesn't change (still default custom color).

So, How do I save that setting?

Thank you.

**Comment by ahrm** [comment, marked as answer]
You can set it in `prefs_user.config` file. See the documentation here: https://sioyek-documentation.readthedocs.io/en/latest/configuration.html#background-color

**Reply by tanreeve** [reply]
Can't I change it through the command palette and then it's automatically saved? or I have to manually edit prefs_user.config ?

**Reply by ahrm** [reply]
You have to manually edit `prefs_user.config`.

**Reply by tanreeve** [reply]
Ok. I get it. thank you for your help. 👍
